From 956c1637a681745bf8833a2acb2543bfea576080 Mon Sep 17 00:00:00 2001
From: Rebecca Swords <rswords@grammatech.com>
Date: Mon, 18 Dec 2017 09:50:21 -0600
Subject: [PATCH] [build] Set SERAPI_COQ_HOME from command line

Use ?= so that SERAPI_COQ_HOME can be set from the command line, and
editing the Makefile is no longer necessary.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 28d6d0d..bcc1524 100644
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ OCB_OPT=-use-ocamlfind -j 4 #-classic-display
 OPAMPATH=$(shell opam config var lib)
 
 # For OPAM-installed coq 8.6.dev
-SERAPI_COQ_HOME=
+SERAPI_COQ_HOME?=
 # SERAPI_COQ_HOME=/home/egallego/external/coq-v8.7/
 
 all: sertop
-- 
2.7.4

